README.md

```diff
@@ -58,11 +59,62 @@ The implementation uses the SQS transport provided by [Symfony Amazon SQS Messen
 so all those features are supported. If you already use that transport, the transition to AWS Lamdba is very easy and 
 should not require any change for dispatching messages. 
 
-> Note: when running Symfony on AWS Lambda, it is not necessary to configure credentials. The AWS client will read them [from environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html#configuration-envvars-runtime) automatically.
+#### Create the SQS queue
+
+You can create the Queue yourself in the Console, write custom Cloudformation 
+or use [Lift's Queue construct](https://github.com/getlift/lift/blob/master/docs/queue.md) that will handle that for you. 
+
+Here is a simple example with Lift, check out the [full documentation](https://github.com/getlift/lift/blob/master/docs/queue.md) for more details.
```

Comment:
Maybe mention that the Lift plugin needs to be installed? (with a link to the README?)

Reply:
Good point. Done :+1: